### PR TITLE
Wglc update

### PR DESCRIPTION
**Pull request by huitema**
Update the outdated references (Issue #20).
Address MT's comments (Issue #21).

### CODE REVIEW

File: draft-ietf-tls-sni-encryption.md [hunk 1]
```diff
@@ -318,14 +319,14 @@ apply just as well for these transports as for TLS over TCP.
 This points to a requirement for SNI Encryption mechanisms to also
 be applicable to non-TCP transports such as DTLS or QUIC.
 
-## Fail to fronting
+## Mitigate detection by TLS proxies
```
Comment (martinthomson) [inline draft-ietf-tls-sni-encryption.md]: Maybe you need to describe the scenario you are concerned about right up front.  That is,

> A TLS proxy is a middlebox that terminates TLS connections from clients for the purposes of monitoring or modifying the content of communications.  This is usually enabled by configuring clients with a trust anchor controlled by the TLS proxy.

Then you can avoid using the gendered Man-in-the-Middle and use "TLS proxy" ;)

**Review by martinthomson — COMMENTED** [review]
What is most interesting about the TLS proxy case is that it sets the bar higher for this feature than we typically have it.  If the TLS proxy is able to install a trust anchor, it can see all communications.  However, this says that we don't want it to learn the SNI.  So it can see everything, except this one thing.

I know that this is somewhat a policy decision, it's a weird design goal.

File: draft-ietf-tls-sni-encryption.md (hunk 1 above)
Comment (huitema) [reply draft-ietf-tls-sni-encryption.md]: In fact, I am considering removing the whole section 3.9, based on the feedback.

Comment (huitema) [reply draft-ietf-tls-sni-encryption.md]: Removing section 3.9

**Pull request merged**